priv/perf/apps/load_test/lib/connection/child_chain.ex

```diff
@@ -1,22 +1,26 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 defmodule LoadTest.Connection.ChildChain do
   @moduledoc """
-  Module that overrides the Tesla middleware with the url in config.
+  Module that overrides the Tesla middleware with the url from config.
   """
 
   alias LoadTest.Connection.Utils
 
   def client() do
-    base_url = Application.get_env(:load_test, :child_chain_url)
-
-    middleware = [
-      Tesla.Middleware.Logger,
-      {Tesla.Middleware.BaseUrl, base_url},
-      {Tesla.Middleware.EncodeJson, engine: Jason},
```

Comment:
@boolafish What was the reason we were using Jason instead of Poison?

Reply:
Ino mentioned that Jason is faster than Poison. During the time of introduction to Jason, it does not break the smoke test that I wrote in here. However, I see it is non-compatible from the PR @pik694 wrote since they are still slightly different in decoding/encoding from the 2 libs.

I don't mind to move back to Poison.

https://github.com/phoenixframework/phoenix/issues/2693#issuecomment-388881650

Reply:
Thanks Andy!